php-practice_1.php

```diff
@@ -1,41 +1,188 @@
+<!--  Q1 変数と文字列 -->
 <?php
-// Q1 変数と文字列
 
+$name = '吉野耕平';
+
+echo '私の名前は「' . $name . '」です。';
 
 // Q2 四則演算
+<?php
+
+$num = 5 * 4;
+
+echo $num . "\n";
+echo $num /= 2;
 
 
 // Q3 日付操作
+<?php
+
+date_default_timezone_set('Asia/Tokyo');
 
+echo date('現在時刻は、Y年m月d日 H時i分s秒です。');
 
 // Q4 条件分岐-1 if文
+<?php
 
+$device = 'mac';
+
+//windows||mac の時
+if ($device === 'windows' || $device === 'mac')
+{
+    echo '使用OSは、' .$device .'です。';
```

Comment:
文字列結合の「.」の**前後**には半角スペースを入れましょう！
他にも何個かあるから探してみてね！
```suggestion
    echo '使用OSは、' . $device . 'です。';